src/prefect/agent/agent.py

```diff
@@ -129,6 +124,21 @@ def __init__(
         self.logger = logger
         self.submitting_flow_runs = set()  # type: Set[str]
 
+        self.logger.debug("Verbose logs enabled")
+        self.logger.debug(f"Environment variables: {self.env_vars}")
```

Comment:
`self.env_vars` might contain templated secrets, e.g., `prefect agent start -e MY_SECRET=$FROM_ENV` so can we instead only log the keys and / or the first 3 characters of the value?